### ISSUE TITLE
Database locking issues for main and job database

### ISSUE THREAD

**Issue by laurenlichtman** [learningequality/kolibri#6448]
### Observed behavior

I am having issues importing new content after a channel has been republished on Studio. The first time I load the content item, I get the Kolibri error message, but then it works after refreshing.

### Expected behavior
Being able to access the content without error. 

### User-facing consequences
Confusing to the user.

### Errors and logs
[kolibri.txt](https://github.com/learningequality/kolibri/files/4041691/kolibri.txt)

```
ERROR 2020-01-09 11:40:17,544 exception Internal Server Error: /api/logger/contentsummarylog/482013c9a9380c0daffd0ee9f0f62ca8/
Traceback (most recent call last):
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/rest_framework/viewsets.py", line 116, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/rest_framework/mixins.py", line 84, in partial_update
    return self.update(request, *args, **kwargs)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/core/logger/api.py", line 85, in update
    self.perform_update(serializer)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/rest_framework/mixins.py", line 80, in perform_update
    serializer.save()
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/rest_framework/serializers.py", line 209, in save
    self.instance = self.update(self.instance, validated_data)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/core/logger/serializers.py", line 260, in update
    instance, validated_data
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/rest_framework/serializers.py", line 986, in update
    instance.save()
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/core/logger/models.py", line 161, in save
    super(ContentSummaryLog, self).save(*args, **kwargs)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/core/auth/models.py", line 222, in save
    super(AbstractFacilityDataModel, self).save(*args, **kwargs)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/morango/models/core.py", line 544, in save
    super(SyncableModel, self).save(*args, **kwargs)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/morango/models/fields/uuids.py", line 123, in save
    super(UUIDModelMixin, self).save(*args, **kwargs)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/django/db/models/base.py", line 808, in save
    force_update=force_update, update_fields=update_fields)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/django/db/models/base.py", line 838, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/django/db/models/base.py", line 905, in _save_table
    forced_update)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/django/db/models/base.py", line 955, in _do_update
    return filtered._update(values) > 0
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/django/db/models/query.py", line 667, in _update
    return query.get_compiler(self.db).execute_sql(CURSOR)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/django/db/models/sql/compiler.py", line 1204, in execute_sql
    cursor = super(SQLUpdateCompiler, self).execute_sql(result_type)
  File "/Users/br/.pex/install/kolibri-0.13.0-py2.py3-none-any.whl.a5423862aa069361191a632fdc39af0bd92e5e19/kolibri-0.13.0-py2.py3-none-any.whl/kolibri/dist/django/db/models/sql/compiler.py", line 899, in execute_sql
    raise original_exception
OperationalError: database is locked
```

### Context
 * Kolibri version: 0.13
 * Operating system: MacOS (was using the .pex installer)
 * Browser: Chrome

**Comment by ivanistheone** [comment]
Some more context on what I observed:


1. Starting from a Kolibri Device with v3 of the channel b4101b8a47714987bf661d3892ed0bfd

2. PUBLISHed version 4 on Studio ([DB file](http://studio.learningequality.org/content/databases/b4101b8a47714987bf661d3892ed0bfd.sqlite3) shows v4)

3. Going through the IMPORT MORE steps to get new materials

4. Shows new version available so click UPDATE (the v4 of the DB gets correctly downloaded to .kolibrihome/content/databases/b4101b8a47714987bf661d3892ed0bfd.sqlite3`)

5. then when it comes to selecting new content...

**Expected:** to be presented with the option update one of the nodes that was changed and option to import one new node added

**Observed:** doesn't show any new content nodes to update/import

**Comment by lyw07** [comment]
I'm not able to reproduce this issue and wondering if I can get some more information on this.
I tried:
1. import a channel with a PDF file and an exercise on Kolibri
2. add a new exercise on Studio and publish the channel
3. on Kolibri, click Manage --> Import More --> Kolibri Studio --> View Changes --> Update Channel

I also tried:
1. import a channel with a PDF file and two exercises (after the update above) on Kolibri
2. Change the metadata of an exercise (title) on Studio and publish the channel
3. on Kolibri, click Manage --> Import More --> Kolibri Studio --> View Changes --> Update Channel

The channel got imported successfully with no database lock errors, but there is actually another error that the content does not get updated at all based on the change made on Studio.

Is there any specific thing I need to do on Studio side to reproduce the error mentioned in this issue? Thank you!
